account_financial_report_qweb/wizard/open_items_wizard.py

```diff
@@ -43,14 +43,39 @@ class OpenItemsReportWizard(models.TransientModel):
     partner_ids = fields.Many2many(
         comodel_name='res.partner',
         string='Filter partners',
+        default=lambda self: self._init_partners(),
+        domain="[('parent_id','=', False)]",
     )
     foreign_currency = fields.Boolean(
         string='Show foreign currency',
+        default=lambda self: self._init_state(),
         help='Display foreign currency for move lines, unless '
              'account currency is not setup through chart of accounts '
              'will display initial and final balance in that currency.'
     )
 
+    def _init_state(self):
+        """methods initialize field due to context from w/ view and w/ current
+        user open wizard
+        """
+        context = self.env.context
+        if context.get('active_model') == 'res.partner' and \
+                self.env.user.has_group('base.group_multi_currency'):
+            return True
+
+    def _init_partners(self):
+        context = self.env.context
+        if context.get('active_ids'):
+            partner_ids = context['active_ids']
+            corp_partners = self.env['res.partner'].search([
```

Comment:
if they are active ids the you should have already been browsed and cached... I'd do:

```py
self.env['res.partner'].browse(active_ids).mapped('parent_id).ids
```